### PR TITLE
Missing comma

### PR DESCRIPTION
**Pull request by krazyeom**
There is missing comma at the end of that line.

### CODE REVIEW

**Comment by vjeux** [comment]
Sorry for not getting to it earlier, can you rebase and i'll merge!

File: Examples/Movies/MovieCell.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ var {
   StyleSheet,
   Text,
   TouchableHighlight,
-  View
+  View,
```
Comment (sebmck) [inline Examples/Movies/MovieCell.js]: This comma is unnecessary.

Comment (brentvatne) [inline Examples/Movies/MovieCell.js]: @sebmck - it is, but as a matter of style an consistency across the project it is used. See: https://github.com/elierotenberg/coding-styles/blob/master/es6.md

Related: if anyone thinks it's worthwhile, I could draft a code style guide for react-native in order to make things more clear cc @vjeux

Comment (vjeux) [inline Examples/Movies/MovieCell.js]: We've had on our todolist to draft a fb coding style guide for a while. Would be nice if you want to do it and follow what we do on react/react native codebase :)

@sebmck we want to always have trailing comma everywhere

**Comment by ccheever** [comment]
@brentvatne are you able to merge code under Examples/ or does that have to be done by FB folks? Seems like there is consensus here and @vjeux has blessed it.

**Comment by brentvatne** [comment]
@vjeux - This is the one I pinged you about on irc, seems pretty small. Would be nice if we could move the `Examples` dir out of the FB sync process with the exception of UIExplorer and SampleApp so I don't have to bother you with these things.

**Comment by vjeux** [comment]
Yeah, need to figure a solution for those

**Pull request closed without merging**